recipes/bareditor/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "bareditor" %}
+{% set version = "0.17" %}
+{% set file_ext = "tar.gz" %}
```

Comment:
I don't think you need the file ext and has variables. Just hard code them below. It makes things a little easier to read.

Reply:
Agreed, done.